src/applications/disability-benefits/526EZ/components/PrivateProviderTreatmentView.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { formatReviewDate } from 'us-forms-system/lib/js/helpers';
+
+export default function PrivateProviderTreatmentView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.treatmentDateRange) {
+    from = formatReviewDate(formData.treatmentDateRange.from);
+    to = formatReviewDate(formData.treatmentDateRange.to);
```

Comment:
Looks like this isn't working for some reason:
![image](https://user-images.githubusercontent.com/12970166/47377799-17b09400-d6ab-11e8-8f9c-8095bde293d0.png)

I did enter the dates and they validated to get me this far.

Reply:
That's weird, you're not getting any error are you? What date did you enter? Or could you slack me the formData? Maybe I can fill it in to check it.

Reply:
Aww man. This one is leaving me scratching my head. I'm getting this: 
![screen shot 2018-10-24 at 4 18 58 pm](https://user-images.githubusercontent.com/11091836/47458845-a1925700-d7a8-11e8-9b53-20fd4cf48f96.png)

I added space before and after the dash, but I noticed the change wasn't in here when you first tested it. I've just updated the file and pushed it again. I'm hoping something just went wonky for a second. Do you mind re-testing?

Reply:
Welllllll, that's odd. Good news: It works! 
![image](https://user-images.githubusercontent.com/12970166/47466132-bd93f980-d7a4-11e8-94da-cfd80e52cf25.png)

:man_shrugging: